src/api/sockets.js

```diff
@@ -19,35 +19,34 @@ function onConnection(socket) {
 
   socket.on('addToQueue', () => {
       if (!waitingRooms.includes(socket)) {
-      waitingRooms.push(socket)
+        waitingRooms.push(socket)
 
         if (waitingRooms.length >= 2) {
-          gameRoom = room += 1
+          console.log('second if')
+          room ++
+          gameRoom = 'room ' + room
           activeRooms[gameRoom] = [waitingRooms.pop(0), waitingRooms.pop(1)]
```

Comment:
Unlike Python, JS `pop` doesn't take an index. It just returns the last element of the array.

As an aside, they also happen one at a time in both languages. So, this would've returned the 1st and 3rd elements of the array. (e.g.  `['a','b','c'] ` would be `['b','c']` after `pop(0)`)